production_ansible/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ansible==2.9.5
+requests==2.22.0
+botocore
```

Comment:
Fix the version for botocore and boto3 dependencies

Reply:
Nice catch! Done on 892d465aa21f80e68017d033b33c59930c12c276.